README.md

```diff
@@ -78,3 +78,8 @@ amce:
 ```
 
 The translator will then pick up all enum classes defined in your translation file.
+
+Example implementation:
+```twig
+{{ constant('Amce\\ExampleBundle\\Entity::DONE') | trans([], 'Amce\\ExampleBundle\\Entity') }} {# Resulting in 'Installation Complete.', if set to English. #}
```

Comment:
Can you put the comment on a new line? Otherwise you'll get a scrollbar which makes it harder to read